src/settings.js

```diff
@@ -58,6 +58,7 @@ const locales = {
   ES_ES: 'es_ES', // Spanish (Spain)
   ES_MX: 'es_MX', // Spanish (Mexico)
   FR_FR: 'fr_FR', // French (France)
+  IT_IT: 'it_IT', // Italian (Italy)
```

Comment:
I don't think we have Italian translations?

Reply:
See DEV-5962

Reply:
See dev ticket